src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -238,8 +246,8 @@ export const TradeInput = (props: CardProps) => {
                 assetId={buyAsset.assetId}
                 assetSymbol={buyAsset.symbol}
                 assetIcon={buyAsset.icon}
-                cryptoAmount={buyAmountAfterFeesCryptoPrecision}
-                fiatAmount={buyAmountAfterFeesUserCurrency}
+                cryptoAmount={positiveOrZero(buyAmountAfterFeesCryptoPrecision).toString()}
```

Comment:
do we need to do `toFixed()` for these ones?

Reply:
It definitely doesn't hurt: https://github.com/shapeshift/web/pull/4917/commits/f7c30799409759f2a6041c6237bfc485a3b4e868